style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+body {
+  display: flex;
```

Comment:
```suggestion
  display: grid;
  place-items: center;
```

Reply:
done!
but whats the advantage of using GRID instead of FLEX?

Reply:
when the same task can be achieved in two line why to write three lines then?